go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/fluxcd/pkg/apis/kustomize v1.1.1
 	github.com/fluxcd/pkg/apis/meta v1.1.2
 	github.com/fluxcd/pkg/runtime v0.42.0
-	github.com/fluxcd/pkg/ssa v0.32.0
+	github.com/fluxcd/pkg/ssa v0.33.1-0.20231026140034-ee4c4a064707
```

Comment:
I propose we first update the controller to Kubernetes 1.28, then come back to this PR and rebase with SSA v0.34.0